test/distributed/_tensor/test_random_ops.py

```diff
@@ -243,7 +243,7 @@ def test_deterministic_uniform_2d(self):
             self.assertEqual(shard_linear_idx, shard_index[self.rank])
 
             # compute local size and offset
-            local_shard_offset = compute_local_offset(
+            _, local_shard_offset = compute_local_shape_and_global_offset(
```

Comment:
@XilunWu Just want to let you know that we are updating this API call in test_random_ops as we planned to remove compute_local_offset() from _utils.py. 

Previously, compute_local_offset gives local_offset. local_offset is different from global_offset if a dimension has been sharded multiple times. More context can be found here: https://github.com/pytorch/pytorch/pull/107996